GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -395,7 +395,7 @@ private void DiffFiles_SelectedIndexChanged(object sender, EventArgs e)
         private void DiffFiles_DoubleClick(object sender, EventArgs e)
         {
             FileStatusItem item = DiffFiles.SelectedItem;
-            if (item?.SecondRevision == null || item.Item == null || !item.Item.IsTracked)
+            if (item?.SecondRevision == null || !item.Item.IsTracked)
```

Comment:
We are 100% certain `item.Item` can never be `null` at this point, right?

Reply:
If it is null, many other actions will be broken
This is when invoking actions on the selections so also item should not be null (but there has been issues...)

The risk with the change is when creating the FileStatusItem where GitStatusItem is null but not used anyway. This is why SecondRevision is [CanBeNull], ViewPullRequestsForm has special handling that never sets or accesses the first/second revision and I prefer to not add a dummy revision.